app/services/i18n/i18n.ts

```diff
@@ -98,7 +98,7 @@ export class I18nService extends PersistentStatefulService<II18nState> implement
 
     // load dictionary if not loaded
     if (!this.loadedDictionaries[locale]) {
-      await this.loadDictionary(this.state.locale);
+      await this.loadDictionary(locale);
```

Comment:
this is the main issue - I have to use `locale` instead `this.state.locale` when app cache is clean